devscripts/changelog_override.json

```diff
@@ -127,8 +127,20 @@
         "short": "[ie] Support multi-period MPD streams (#6654)",
         "authors": ["alard", "pukkandan"]
     },
+    {
+        "action": "change",
+        "when": "aa7e9ae4f48276bd5d0173966c77db9484f65a0a",
+        "short": "[ie/xvideos] Support new URL format (#9502)",
+        "authors": ["sta1us"]
+    },
     {
         "action": "remove",
         "when": "22e4dfacb61f62dfbb3eb41b31c7b69ba1059b80"
+    },
+    {
+        "action": "change",
+        "when": "e3a3ed8a981d9395c4859b6ef56cd02bc3148db2",
+        "short": "[cleanup:ie] No `from` stdlib imports in extractors",
```

Comment:
Pretty sure `cleanup, ie` is supposed to work 🤔

Reply:
no, since they are different depths. Full form is `misc/cleanup` and `extractor` and different depths cannot be merged. Ive asked about this before and since no conflict resolving strategy was proposed I did not implement any